src/main/docker/Dockerfile

```
@@ -27,20 +27,26 @@ ENV TZ=Etc/UTC \
     PATH="/opt/java/openjdk/bin:${PATH}" \
     LANG=C.UTF-8
 
-COPY --from=jre-build /opt/java/openjdk $JAVA_HOME
-
-# Copy the compiled WAR to the application directory created above
-# Automatically creates the $APP_DIR directory
-COPY ./target/${WAR_FILENAME} ${APP_DIR}
-
-# Create the directory where Dependency-Track will store its data (${DATA_DIR}) and the external library directory (${EXTLIB_DIR})
+# Create the directories where the WAR will be deployed to (${APP_DIR}) and Dependency-Track will store its data (${DATA_DIR})
 # Create a user and assign home directory to a ${DATA_DIR}
 # Ensure UID 1000 & GID 1000 own all the needed directories
-RUN mkdir -p -m 770 ${DATA_DIR} \
+RUN mkdir -p -m 770 ${APP_DIR} \
+    && mkdir -p -m 770 ${DATA_DIR} \
     && addgroup --system --gid ${GID} dtrack || true \
     && adduser --system --disabled-login --ingroup dtrack --no-create-home --home ${DATA_DIR} --gecos "dtrack user" --shell /bin/false --uid ${UID} dtrack || true \
     && chown -R dtrack:0 ${DATA_DIR} ${APP_DIR} \
-    && chmod -R g=u ${DATA_DIR} ${APP_DIR}
+    && chmod -R g=u ${DATA_DIR} ${APP_DIR} \
+    \
+    # Install wget for health check
+    && apt update \
+    && DEBIAN_FRONTEND=noninteractive apt install -y wget \
```

```suggestion
    && apt-get -yqq update \
    && DEBIAN_FRONTEND=noninteractive apt-get install -yqq wget \
```
This is personal preference, but 
1. `apt` is 'non script-able' / not recomended to be run in scripts, where `apt-get` is IIRC
2. its quite nosy IMHO so `qq` (still nosy but less)